app/controllers/concerns/decidim/proposals/orderable.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Decidim
+  module Proposals
+    # Common logic to ordering resources
+    module Orderable
+      extend ActiveSupport::Concern
+
+      included do
+        include Decidim::Orderable
+
+        private
+
+        # Available orders based on enabled settings
+        def available_orders
+          @available_orders ||= begin
+            available_orders = %w(random recent)
+            available_orders << "most_equitable"
```

Comment:
ℹ️ Kind of sucks needing to copy this whole module in from the proposals gem just to add three lines of code.

Reply:
Suggestion: This could be combined with the above line, if you wanted to make it more compact :)